From: Tobias Geerinckx-Rice <me@tobias.gr>
Date: Sun, 26 Aug 2018 17:24:42 +0200
Subject: [PATCH] snappy: Add O2 flag in CmakeLists.txt.

Use ‘-O2’ optimisation when building with CMake, as is already done when
using the Makefile. This patch was copied verbatim from the Snappy
mailing list[0].

[0]: <snappy-compression@googlegroups.com>

From 903c72fb29b2db07b4abc38a5feb83d88f739d80 Mon Sep 17 00:00:00 2001
From: huangwenjun <huangwenjun-hf@loongson.cn>
Date: Fri, 10 Aug 2018 17:17:35 +0800
Subject: [PATCH] Add O2 optimize flag in CmakeLists.txt.

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 62ecd09..29e0cdc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,9 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
+
 # BUILD_SHARED_LIBS is a standard CMake variable, but we declare it here to make
 # it prominent in the GUI.
 option(BUILD_SHARED_LIBS "Build shared libraries(DLLs)." OFF)
-- 
2.1.0

